guardian/master-to-main
# chore: Bump dependencies

## What does this change?

Following the conversion to ESM, we are able to bump `chalk` to latest (v5 is ESM only) and I've bumped all the other deps as well so we're fully up-to-date for a release.

We are now using Node 18. 

## How to test

Run `yarn build`, `yarn test` and `yarn run:dev` on this branch and see that everything is working as expected.

## Have we considered potential risks?

Are there any risks to updating the version of node to v18?
